minvws/nl-kat-coordination
# Support valid_time and the like for queries in xtdb tools

### Changes
Support valid_time and the like for queries in xtdb tools

### Issue link
N/A

### Demo
Make a setup with some objects.

Verify there are some objects:
`octopoes/tools/xtdb-cli.py query '{:query {:find [(pull ?var [*])] :where [[?var :object_type]]}}' | jq -C | less -R`

Verify there weren't any objects 91 years ago (you can place other dates here):
`octopoes/tools/xtdb-cli.py query --valid-time 1933-2-27 '{:query {:find [(pull ?var [*])] :where [[?var :object_type]]}}' | jq -C | less -R`

### QA notes
Run the demo; perhaps change valid-time value to something around the creation of the objects so you can see the history.

---

### Code Checklist







---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
